modules/timeline.xql

```diff
@@ -27,13 +28,23 @@ for $entry in $entries
 let $dateStr := ($entry/atom:updated, $entry/atom:published)[1]
 let $date := xs:dateTime($dateStr/text())
 let $age := current-dateTime() - $date
+let $minutesSince := minutes-from-duration($age)
+let $hoursSince := hours-from-duration($age)
+let $daysSince := days-from-duration($age)
 let $path := $local:WIKI_ROOT ||substring-after($entry/atom:link[@type eq 'blog']/@href, "/db/apps/wiki/data")
 let $title := $entry/atom:title/text()
-let $abstract := subsequence($entry/atom:content//xhtml:div/xhtml:p[1],1,1)
+let $abstract := subsequence($entry//xhtml:p[1])
 let $image := subsequence($entry/atom:content//xhtml:img[1]/@src,1,1)
 
-let $yeah := httpclient:head(xs:anyURI(concat($path,"/",$image)),false(),())
-let $checkresult := data($yeah/@statusCode) = 200
+let $header := 
+    http:send-request(
+      <http:request method='head' />,
```

Comment:
Another observation: I think we could drop this HTTP HEAD request on the images. If the image is referenced in the Atom source, we can be confident that it's there. Let the client make the request.

Reply:
I do not know enough of the current state of data nor of the history of this website to answer that ;)

Reply:
But it sounds reasonable to assume that images are there if they are referenced in the atom entry